src/qibolab/__init__.py

```diff
@@ -79,3 +79,47 @@ def execute_qasm(circuit: str, platform, runcard=None, initial_state=None, nshot
     return QibolabBackend(platform, runcard).execute_circuit(
         circuit, initial_state=initial_state, nshots=nshots
     )
+
+
+def get_available_platforms() -> list[str]:
+    """Returns the platforms found in the $QIBOLAB_PLATFORMS directory."""
+    return [
+        d.name
+        for d in get_platforms_path().iterdir()
+        if d.is_dir() and not (d.name.startswith("_") or d.name.startswith("."))
+    ]
+
+
+class MetaBackend:
```

Comment:
This is more to @alecandido, who I believe proposed this. There is an inheritance/dependency from qibo here that we are not declaring explicitly, namely qibo expects that `MetaBackend` defines the `load` method. It works in practice, but in principle it is still a bit cyclic: qibo imports this `MetaBackend` from qibolab using `importlib` and qibolab still defines a qibo class even though we don't explicitly inherit it.

Reply:
It is as cyclic as the `Backend` itself. In that case, `qibolab` imports `qibo` to inherit the `AbstractBackend`, and then `qibo` imports `qibolab` within the `construct_backend` function.

Here we're avoiding the inheritance, and we rely on a convention, but the dynamical import still takes the same role of the scoped import (actually, I'm not sure whether we really need a dynamical import, I should review the PR in Qibo).

Here the problem is exactly the Qibojit ones (with the only benefit that Qibolab is not tested in Qibo), so the best solution I have to propose is to wait for `qibo-core`...